### PR TITLE
Undoing/redoing consecutive carriage returns

### PR DESCRIPTION
**Pull request by aaxu**
This doesn't work for when the carriage returns are auto-indented by a colon because that adds a new change to the redo chain and I need to figure out a way to work around that. Need more usage to see if it works better than the current method of undoing/redoing

### CODE REVIEW

**Comment by aaxu** [comment]
@dschuyler When you created the commands.py file, did you forget to delete them from the text_buffer.py file? There is duplicate code and I'm not really sure what the commands.py file is for. I only made the changes to text_buffer.

**Comment by dschuyler** [comment]
I intended to split text_buffer into two files. Part way through that process I realized that might mess you up and I didn't want to do that to you. So I started wondering about how to announce such refactor intentions. ...and I didn't finish. It's a whoops to have left it half done. Sorry. Thanks for asking.

**Comment by aaxu** [comment]
Oh okay thanks for the clarification. I don't really mind. You can go ahead and do that and I'll update this PR if that's more convenient for you.

**Comment by dschuyler** [comment]
Sorry I've been slow to respond this week. I'll try to look at PRs on Friday.

**Comment by dschuyler** [comment]
I decided on the file name actions.py rather than commands.py and committed the code move.

File: app/text_buffer.py [hunk 1]
```diff
@@ -687,6 +707,7 @@ def applyDocumentUpdate(self, data):
 
   def findCurrentPattern(self, direction):
     localRe = self.findRe
+    offset = self.penCol + direction
```
Comment (dschuyler) [inline app/text_buffer.py]: The remainder of this file looks like changes from #32.

File: app/text_buffer.py [hunk 2]
```diff
@@ -172,9 +170,31 @@ def cursorLeft(self):
     self.selectionNone()
     self.cursorMoveLeft()
 
+  def getCursorMove(self, rowDelta, colDelta):
+    return self.getCursorMoveAndMark(rowDelta, colDelta, 0, 0, 0)
+
   def cursorMove(self, rowDelta, colDelta):
     self.cursorMoveAndMark(rowDelta, colDelta, 0, 0, 0)
 
+  def getCursorMoveAndMark(self, rowDelta, colDelta, markRowDelta,
```
Comment (dschuyler) [inline app/text_buffer.py]: It looks like much of the contents of cursorMoveAndMark() can be replaced with a a call to getCursorMoveAndMark(). Does that work?

Comment (aaxu) [inline app/text_buffer.py]: Oh yeah, I missed that! Yeah definitely.

File: app/text_buffer.py (hunk 1 above)
Comment (aaxu) [inline app/text_buffer.py]: Oh woopsies. Thanks for the catch

**Comment by aaxu** [comment]
Requested changes done!

**Pull request merged**